packages/cli/bin/bundle.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @shopify/cli/specific-imports-in-bootstrap-code */
+/* eslint-disable import/no-extraneous-dependencies */
+import cleanBundledDependencies from '../../../bin/bundling/clean-bundled-dependencies.js'
+import ShopifyStacktraceyPlugin from '../../../bin/bundling/esbuild-plugin-stacktracey.js'
+import ShopifyVSCodePlugin from '../../../bin/bundling/esbuild-plugin-vscode.js'
+import {build as esBuild} from 'esbuild'
+import requireResolvePlugin from '@chialab/esbuild-plugin-require-resolve'
+import {copy} from 'esbuild-plugin-copy'
+import glob from 'fast-glob'
+
+const external = [
+  // react-devtools-core is a dev dependency, no need to bundle it but throws errors if not included here.
+  'react-devtools-core',
+  // esbuild can't be bundled per design
+  'esbuild',
+]
+
+// yoga wasm file is not bundled by esbuild, so we need to copy it manually
+const yogafile = glob.sync('../../node_modules/.pnpm/**/yoga.wasm')[0]
+
+esBuild({
```

Comment:
I guess a subsequent PR will add a GH action that triggers (and checks??) bundling?

Reply:
A subsequent script will add the shipit config to trigger the bundling.
I didn't add a GH action to validate that bundling works but will do :)